pages/_document.tsx

```diff
@@ -0,0 +1,45 @@
+/**********************************************************
+ * Embed the Google Analytics script in the common Head of all pages.
```

Comment:
ここも、GAのためだけのものではありません。

https://nextjs.org/docs/advanced-features/custom-app

> A custom Document is commonly used to augment your application's <html> and <body> tags. This is necessary because Next.js pages skip the definition of the surrounding document's markup.

Reply:
修正しました